solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder/src/main/java/com/solace/spring/cloud/stream/binder/config/HealthIndicatorsConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package com.solace.spring.cloud.stream.binder.config;
+
+import com.solace.spring.cloud.stream.binder.health.contributors.BindingsHealthContributor;
+import com.solace.spring.cloud.stream.binder.health.contributors.ConnectionHealthContributor;
+import com.solace.spring.cloud.stream.binder.health.handlers.SolaceSessionEventHandler;
+import com.solace.spring.cloud.stream.binder.health.indicators.SessionHealthIndicator;
+import com.solace.spring.cloud.stream.binder.properties.SolaceFlowHealthProperties;
+import com.solace.spring.cloud.stream.binder.properties.SolaceSessionHealthProperties;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.boot.actuate.autoconfigure.health.ConditionalOnEnabledHealthIndicator;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnClass(name = "org.springframework.boot.actuate.health.HealthIndicator")
+@ConditionalOnEnabledHealthIndicator("binders")
+@EnableConfigurationProperties({SolaceSessionHealthProperties.class, SolaceFlowHealthProperties.class})
+public class HealthIndicatorsConfiguration {
```

Comment:
Another minor naming thing, but could we rename this to something like `SolaceBinderHealthConfiguration`? Just so devs know that this is specific to Solace...